src/components/DetailsContainer/DetailsContainer.js

```diff
@@ -1,48 +1,98 @@
-import React from "react";
+import React, { useState , useEffect } from 'react';
 import './DetailsContainer.css';
-import arrow from '../../assets/arrow.png';
 import { Link } from 'react-router-dom';
+import Header from '../Header/Header';
+import colorado from '../../assets/colorado-flag.webp';
+import { getSingleTree } from '../../apiCalls';
 
-const DetailsContainer = ({selectedTree, clearSelectedTree}) => {
 
-   console.log('details container selected tree',selectedTree)
+const DetailsContainer = ({selectedTree, clearSelectedTree, clearZipTrees, match}) => {
+  window.scroll(0,0);
+  console.log('match line 11', match)
+    const [myTree , setMyTree] = useState({})
 
-   const createHTML = () => {
-      return {__html: selectedTree.overview_description};
-   }
+    useEffect(() => {
+      console.log('match',match)
+      getSingleTree(match)
+        .then(data => {
+          setMyTree(data[0])
```

Comment:
Great job pulling out the first index in the state so that the selectedTree in the return did not need to be changed to [0] manually 🕺